src/layouts/sidebar-sidecar/types.ts

```diff
@@ -23,25 +18,5 @@ interface BaseProps {
 	versions?: VersionSelectItem[]
 	showScrollProgress?: boolean
 	mainWidth?: 'wide' | 'narrow'
+	sidecarSlot?: React.ReactNode
```

Comment:
💭 Since we're already importing `ReactElement` from `'react'`, could we import `ReactNode` too and use that here and for `children`?

Reply:
For sure! Updated 👍